cli/src/print.rs

```diff
@@ -60,3 +60,37 @@ pub(crate) fn print_create_android_success_message() {
     print_green_bold("This will open the Android project in Android Studio.".to_string());
     print_footer_message();
 }
+
+pub(crate) fn print_create_flutter_success_message() {
+    print_green_bold("Flutter template created successfully!".to_string());
+    println!();
+    print_green_bold("Next steps:".to_string());
+    println!();
+    print_green_bold(
+        "  You can now use the following command to open the app in Android Studio:".to_string(),
+    );
+    println!();
+    print_bold(r"    open flutter-app -a Android\ Studio ".to_string());
+    println!();
+    print_green_bold("  or VS Code::".to_string());
+    println!();
+    print_bold(r"    code flutter-app".to_string());
```

Comment:
I think we can direct them to read the README in the `flutter-app` foler
e.g. please follow the `fluttter-app/README.md` to start the app and replace your circuit
Or link to https://zkmopro.org/docs/setup/flutter-setup
It will be more clear for beginners